js/storage.md

```diff
@@ -61,13 +61,16 @@ Amplify.configure({
         userPoolWebClientId: 'XX-XXXX-X_abcd1234', //OPTIONAL - Amazon Cognito Web Client ID
     },
     Storage: {
-        bucket: '', //REQUIRED -  Amazon S3 bucket
-        region: 'XX-XXXX-X', //OPTIONAL -  Amazon service region
+        AWSS3: {
+            bucket: '', //REQUIRED -  Amazon S3 bucket
+            region: 'XX-XXXX-X', //OPTIONAL -  Amazon service region
+        }
     }
 });
 
 ```
 
+## Using Amazon S3
```

Comment:
I think we could add instructions for other xyz provider and how that will configure those as well on other providers section. Analytics has a section for that https://aws-amplify.github.io/docs/js/analytics#using-a-custom-plugin

Reply:
Added the custom plugin section.